lib/support/live_select_web/live/showcase_live.html.heex

```diff
@@ -71,6 +71,10 @@
               <span class="label-text mr-1">Disabled:&nbsp;</span>
               {checkbox(@settings_form, :disabled, class: "toggle")}
             <% end %>
+            <%= label class: "label cursor-pointer" do %>
+              <span class="label-text mr-1">Custom option HTML:&nbsp;</span>
+              <%= checkbox(@settings_form, :custom_option_html, class: "toggle") %>
```

Comment:
I love that you went out of the way to create this toggle to show how to style the options as checkboxes!

Can we please rename it to something more specific like "Options styles as checkboxes" ?